### ISSUE TITLE
Transitioner Example

### ISSUE THREAD

**Issue by brennanerbezniksc** [react-community/react-navigation#250]
Can you write a full example/file of how to use Transitioner? There are some missing pieces in the current doc that make it hard to follow along.

**Comment by grabbou** [comment]
Have you managed to get it up and running thought? If yes, it would be great if you could help with contributing!

**Comment by brennanerbezniksc** [comment]
I've managed to get something up and running. I'm not sure what it is though. I've used the code from the Transitioner docs and some of the methods don't seem to exist. `router.getComponent` throws an error, for example. Any ideas?

**Comment by arnsa** [comment]
+1, need this badly. I have no idea how to use transitioner right now, the docs are too confusing :(. Could anybody help?

**Comment by getnashty** [comment]
+1, I'll try soon!

**Comment by Jose4gg** [comment]
I don't know how to use transitioner yet...Please help

**Comment by studiobrain** [comment]
I have been trying to get a Transitoner to work with custom tabs for days.  Can anyone provide a working example of the Transitoner code similar to: https://github.com/react-community/react-navigation/blob/master/examples/NavigationPlayground/js/CustomTabs.js ?

**Comment by aranda-adapptor** [comment]
+1 for a working example and improved docs :)

I noticed @lintonye has an awesome series of blog posts and the source code is [available](https://github.com/lintonye/react-native-diary/tree/master/transitions), but it is pointing at react-navigation master branch, which seems dangerous given the API is transitioning.

**Comment by kuznetsov0209** [comment]
Working example of the Transitioner:
https://github.com/kuznetsov0209/react-navigation-transitioner/blob/master/src/App.js

**Comment by spencercarli** [comment]
Example added to NavigationPlayground via #2412!
